horace_core/utilities/split_vector.m

```diff
@@ -0,0 +1,100 @@
+function [chunks, idxs] = split_vector(vector, max_chunk_sum)
```

Comment:
is `vector` a good variable name? vector of what? it's a keyword in a lot of languages which may lead to confusion.